tethysapp/water_data_explorer/public/css/main.css

```diff
@@ -5,6 +5,10 @@
   width: 50%;
 }
 
+.hidden {
+  display: none;
+}
+
```

Comment:
Looks like we added the `hidden` rule. Remove it and then replace `hidden` class with `d-none` like with the other apps.

Reply:
When I do a search on hidden, I get 3359 results in 457 files....  I thought that was extreme enough to handle it with a css rule.  If you prefer, I'll go through the 3k results, though.

Reply:
That makes sense. Let's leave the rule then.